test/ComposableCoW.twap.t.sol

```diff
@@ -225,6 +231,40 @@ contract ComposableCoWTwapTest is BaseComposableCoWTest {
         twap.getTradeableOrder(address(0), address(0), bytes32(0), abi.encode(o), bytes(""));
     }
 
+    function test_getTradeableOrder_e2e_fuzz_WithContext(uint32 _blockTimestamp, uint256 currentTime) public {
+        // guard against overflows
+        vm.assume(_blockTimestamp < type(uint32).max);
+        vm.assume(currentTime < type(uint32).max);
+        // guard against revert before start
+        vm.assume(_blockTimestamp < currentTime);
```

Comment:
Not necessarily as a fuzz test, but might be nice to have one test case showing that blockTimestamp is actually used a start time (technically startTime 0 would have worked in the old codebase wouldn't it?)

Reply:
Absolute fair - incoming!